### ISSUE TITLE
Make FAH partially open source

### ISSUE THREAD

**Issue by informatorius** [FoldingAtHome/fah-issues#1224]
Make FAH partially open source. The FAHclient or FAHcontrol could be partially open source, so others can help to improve the software. The private parts and FAHcores can still stay private. e.g. when FAHcontrol has a problem with themes then anybody could look into this and maybe fix it and give a pull request. Still FAH staff decides if you want that pull request included or not in next release version.

**Comment by jcoffland** [comment]
We plan to develop a new Open-Source client.  It's actually possible to develop such a client now.  The NaCl client is Open-Source and uses a JSON API that could also be used as a desktop client.  We need to document it though.

**Comment by bb30994** [comment]
I presume that means that the (in-)efficiency of a JSON client isn't really a factor ... which makes some sense since the client doesn't use anywhere as much resources as the FAHCore.

Would we be generating a problem in supporting a variety of JSON revisions on multiple platforms?

**Comment by dmc5179** [comment]
Any updates on this?

**Comment by bb30994** [comment]
A new FAHClient is under active development, fixing a number of bugs.  I don't know if it will be entirely ready to be OpenSourced, but it will certainly be a better starting point for whatever happens next.

**Comment by ffissore** [comment]
Duplicate of #1224

**Comment by dmc5179** [comment]
@ffissore isn't this issue #1224 ?

**Comment by ffissore** [comment]
yeah @dmc5179 sorry: the comment was intended for issue #1338, which @shorttack has closed in the meantime :+1:
